src/content.js

```diff
@@ -211,20 +215,21 @@ function handlePrCreatePage() {
                 async: false,
                 success: function(result) {
                     $('input#pull_request_title').val('['+ticketNumber.toUpperCase()+'] ' + result.fields.summary);
-                    /*ticketDescription = result.fields.description
-                        .replace('h1.', '# ')
-                        .replace('h2.', '## ')
-                        .replace('h3.', '### ')
-                        .replace('h4.', '#### ')
-                        .replace('h5.', '##### ')
-                        .replace('h6.', '###### ')
-                        .replace('{code}', '```' + NL)
-                        .replace('{{', '``')
-                        .replace('}}', '``');*/
+
+                    var startString = 'h3. Acceptatiecriteria';
```

Comment:
I get that the tickets use the word "Acceptatiecriteria" but its Dutch only now. Could you add a set of trigger-words? So we could add English, Dutch, maybe some abbreviations, etc.